composer.json

```diff
@@ -20,6 +20,7 @@
   },
   "require-dev": {
     "automattic/vipwpcs": "^2.1",
+    "civicrm/composer-downloads-plugin": "^2.1",
```

Comment:
Nice! I assume this is to install this regardless of PHP version set in `composer.json`? Maybe worth for us to change that in Site Kit too.

Reply:
Yep! :-)